src/ontology/cl-edit.owl

```diff
@@ -29524,6 +29525,16 @@ AnnotationAssertion(rdfs:label obo:CL_4028006 "alveolar type 2 fibroblast cell")
 SubClassOf(obo:CL_4028006 obo:CL_0002241)
 SubClassOf(obo:CL_4028006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002299))
 
+# Class: obo:CL_4030000 (choroidal melanocyte)
+
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_4030000 "http://orcid.org/0000-0002-1773-2692")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31819095"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:34082079"^^xsd:string) obo:IAO_0000115 obo:CL_4030000 "Choroidal melanocytes are melanin-producing cells in the uvea of the human eye. Their cellular functions and extracellular interactions are distinct to skin melanocytes. They are involved in photoprotection, regulation of oxidative damage and immune responses. Choroidal melanocytes might have both neuroectodermal as well as mesectodermal origins.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030000 "http://orcid.org/0000-0002-0819-0473")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_4030000 "2022-01-20T12:59:06Z"^^xsd:dateTime)
```

Comment:
not sure we use date stamps in CL - at least I don't. Don't see it around much and I guess trying to follow convention is ideal.

Reply:
I see some terms with a "creation date" annotation. If I change the annotation to "creation date", would that be better? Or do you propose eliminating creation date going forward?

Reply:
Bit shocked we're still arguing over this.  Looks like Bradley has the [setup specified in the Mondo tutorial](https://oboacademy.github.io/obook/lesson/contributing_to_obo_ontologies/#setting-preferences-for-new-entities-metadata).  I'm still using creation_date - as this is what we taught in the last CL workshop.  I don't want to derail work by worrying about it, but we should aim for a consensus on the standard, we can then retrofit for consistency.  Perhaps ping Nicole and Nico for their opinion on this?

Reply:
But we shouldn't worry about it for this PR